### PR TITLE
[TASK] Fix deprecations with TYPO3 8.x

### PR DESCRIPTION
**Pull request by thomashohn**
Removed $GLOBALS['PATH_solr'] , $GLOBALS['PATHrel_solr'], ExtensionManagementUtility::extRelPath('solr') and temp fix for #948

### CODE REVIEW

File: ext_localconf.php [hunk 1]
```diff
@@ -3,14 +3,12 @@
     die('Access denied.');
 }
 
-$GLOBALS['PATH_solr'] = \TYPO3\CMS\Core\Utility\ExtensionManagementUtility::extPath('solr');
```
Comment (irnnr) [inline ext_localconf.php]: Why?

**Comment by irnnr** [comment]
What's the point of removing those shortcut globals?

**Comment by thomashohn** [comment]
First of all - they are not really used at that many places and there they are used the "EXT:solr/..." notation can be used instead. If you have good reasons for keeping them - we can keep them :-)

**Comment by thomashohn** [comment]
@irnnr Any news her?

**Comment by timohund** [comment]
@thomashohn Thanks for your pull request! I'll try to have a look on it today. In general it is good and important that we fix it before the release for 8 LTS. Regarding the globals we also need to check solrfal and solrfluid if they are used there.

**Comment by timohund** [comment]
@thomashohn @irnnr for me this looks good. I've created an Issue for version 7.0 to drop the compatibility code for TYPO3 7.6 LTS (#950)

**Comment by timohund** [comment]
@thomashohn I've checked the patch with 7.6 LTS and the index inspector does not show up.

**Comment by thomashohn** [comment]
@timohund Should work with 7.6.15 apparently EXT:extension name is not fully supported in addJS in 7.6.15

**Pull request merged**